cmd/common.go

```diff
@@ -237,6 +238,64 @@ func loadConfig(nodetype check.NodeType) string {
 	return filepath.Join(path, file)
 }
 
+func mapToCISVersion(kubeToCISMap map[string]string, kv string) string {
+	cisVersion, found := kubeToCISMap[kv]
```

Comment:
```suggestion
	benchmark, found := kubeToBenchmark[kv]
```

and similar throughout